common/utils/utils.go

```diff
@@ -53,3 +53,9 @@ func EnsureTrailingSlash(path *string) {
 
 // helper func to package strings up nicely for protobuf
 func ProtoString(s string) *string { return &s }
+
+func RemoveTrailingSlash(path *string) {
```

Comment:
To be checked if there is an `os.path` function that can already do this for us.

Reply:
Change it to `filepath.Clean()`